lametro/models.py

```diff
@@ -740,8 +740,15 @@ def related_files(self):
 
         for item in agenda_items:
             for entity in item.related_entities.filter(bill__isnull=False):
-                bill_packet = BillPacket(bill=entity.bill)
-                related.extend(bill_packet.related_files)
+                try:
+                    related_bill = entity.bill
+                except LAMetroBill.DoesNotExist:
+                    # If this exception occurs, the related bill is private.
+                    # Skip it.
```

Comment:
could this also happen for test bills?

Reply:
Good catch – yes! It can happen if the bill does not meet the criteria for display, only one of which is because it's private:

https://github.com/datamade/la-metro-councilmatic/blob/763d58553f4dd2d2041d8abd6e9e12a6fa25346d/lametro/models.py#L51-L58

I'll clarify this comment.